theSrc/scripts/LabeledScatter.js

```diff
@@ -161,37 +160,14 @@ class LabeledScatter {
           console.log('rhtmlLabeledScatter: resize timeout')
 
           const resizeParams = this.resizeStack.pop()
-          const el = resizeParams[0]
           const width = resizeParams[1]
           const height = resizeParams[2]
           this.resizeStack = []
 
           this.width = width
           this.height = height
 
-          if (typeof this.plot === 'undefined') {
-            this.draw()
-          } else {
-            d3.select('.plot-container').remove()
```

Comment:
The resize test was failing previously due to this line. I'm removing all this code and just calling `this.draw()` on resize. If there are any issues with this we'll find it once combined scatterplot is in Displayr.